### PR TITLE
Approving creds is removed

### LINKED ISSUE #1: approving creds with the container token doesn't work
- Remove approving creds. Approving new creds with the container-provided token doesn't work and instead must be done with "system:admin" creds.
- Do not persist new creds if they are not valid.

### LINKED ISSUE #3: CI: 6 hours timeout on kubectl version during the image build
https://github.com/nuvlaedge/kubernetes-credentials-manager/actions/runs/4553538199/jobs/8030218799

and

https://github.com/nuvlaedge/kubernetes-credentials-manager/actions/runs/4553538008/jobs/8030218492

```
Wed, 29 Mar 2023 11:38:39 GMT #7 1.982 + kubectl version --client=true
Wed, 29 Mar 2023 17:38:22 GMT Error: The operation was canceled.
```

### PR DESCRIPTION
**Pull request by konstan**
- Removed approving creds. Approving new creds with the container-provided 
  token doesn't work and instead must be done with "system:admin" creds.
- Do not persist new creds if they are not valid. 

Fixes #1 
Fixes #3

### CODE REVIEW

**Comment by konstan** [comment]
@schaubl PR is ready for review.

File: Dockerfile [hunk 1]
```diff
@@ -35,7 +35,7 @@ RUN set -eux; \
     	curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/${kubectlArch}/kubectl && \
     	chmod +x ./kubectl && \
     	mv ./kubectl /usr/local/bin/kubectl && \
-    	kubectl version --client=true
+        which kubectl
 
 EXPOSE 8001
```
Comment (giovannibianco) [inline Dockerfile]: We expose port 8001 here... but in the kubernetes-credential-manager.sh the "kubectl proxy" command is removed ... that exposes port 8001. Do we need this still?

File: code/kubernetes-credential-manager.sh [hunk 2]
```diff
@@ -1,27 +1,41 @@
 #!/bin/sh -xe
 
-kubectl proxy &
```
Comment (giovannibianco) [inline code/kubernetes-credential-manager.sh:3]: See comment above in the Docker file

File: Dockerfile (hunk 1 above)
Comment (konstan) [reply Dockerfile]: Agreed. Not needed. Removing.

**Pull request merged**